packages/immutable-rootfs/30cos-immutable-rootfs/module-setup.sh

```diff
@@ -5,9 +5,9 @@ check() {
     return 255
 }
 
-# called by dracut
+# called by dracut 
 depends() {
-    echo rootfs-block dm
+    echo rootfs-block dm fs-lib
```

Comment:
If fs-lib is here, then it should not be needed in `dracut.conf` file.

Reply:
ouch good catch, I think it slipped in while I was testing